### ISSUE TITLE
Segmentation Fault openvdb tools

### ISSUE THREAD

**Issue by Aadi279** [microsoft/vcpkg#6332]
Running openvdb_viewer on windows causes seg fault

**Comment by wangli28** [comment]
Hi @Aadi279, can you provide more information? How did you use this tools? And what information did you get?
Thanks.

**Comment by Aadi279** [comment]
Hi @wangli28 

I installed vdb first ( vcpkg install openvdb) followed by the tools ( vcpkg install openvdb[tools] ).

The necessary pre-req libraries were also installed first (vcpkg install zlib boost openexr tbb ). 

I then integrated the install . 

To execute the vdb_viewer , I cd into the folder with the executable vcpkg/installed/x86-windows/tools/openvdb .
 I copy over an example model file ( tried many from the website https://www.openvdb.org/download/  - buddha.vdb for example here ) and tried running the command ./vdb_view buddha.vdb and it results in a segmentation fault.
